app/views/report_mailer/tables_report.html.erb

```diff
@@ -10,7 +10,7 @@
         height: 21,
       ) %>
 </header>
-<hr class="height-05 margin-bottom-4 border-transparent">
+<hr class="height-05 margin-bottom-4 border-transparent bg-secondary">
```

Comment:
Should this also be applied on the `<hr />` on line 50?

Reply:
Yes it should! Thank you.